examples/common/utilities.adb

```diff
@@ -11,6 +11,13 @@ with GL.Types.Colors;
 
 package body Utilities is
 
+   procedure Clear_All (Colour : GL.Types.Colors.Color) is
+   begin
+        GL.Buffers.Clear ((True, True, True, True));
+        GL.Buffers.Set_Color_Clear_Value (Colour);
```

Comment:
Yes.
I checked the OpenGL documentation and now understand the process.
Set_Color_Clear_Value sets the value to which a buffer should be set when cleared.
Clear implements glclear which "clears" selected values to the previously selected value set by Set_Color_Clear_Value .